modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -72,6 +72,7 @@ export default function ViewObjectDefinitions({
 	objectDefinitionsFDSName,
 	objectDefinitionsStorageTypes,
 	objectFolderPermissionsURL,
+	sorts,
```

Comment:
Do we need this here? If client code was not using the `sorting` prop in their logic before, this should not be needed.

Reply:
Interesting...this file seems to have been deeply refactored:

![image](https://github.com/liferay-frontend/liferay-portal/assets/20504176/b637dede-571d-4f05-a6ab-b09feeac5d24)

so, we can safely remove the sorts property